Nemt/nemt.h

```diff
@@ -5,10 +5,14 @@
 #include <Pixy.h>
 
 #define CAMERA_CENTER 160
+#define NOT_FOUND 404
+#define DEFAULT_COUNTER_LIMIT 50
+#define CAMERA_WIDTH 320
+#define CAMERA_HORIZONTAL_ANGLE 70
```

Comment:
Also, please leave out your philosophical / political and other opinions out of the code review ;-)